fetch.bs

```diff
@@ -4103,6 +4103,23 @@ steps:
    <var>processBody</var>, <var>processBodyError</var>, and <var>fetchParams</var>'s
    <a for="fetch params">task destination</a>.
   </ol>
+
+ <li><p>If <var>fetchParams</var>'s <a for="fetch params">process response done</a> is non-null, and
```

Comment:
Nit: li has multiple children so p needs to be on a newline and indented. ol below also needs indentation.

Reply:
It doesn't have multiple children in the new commit.